centinel/views.py

```diff
@@ -326,12 +326,17 @@ def get_system_status():
     doesn't reveal anything important (e.g. IP address, username, etc.).
     The list is shuffled each time so that numbers are randomly assigned.
 
+    Note: we don't display clients who have the dont_display column set to 1/True
+
     """
     clients = Client.query.all()
     random.shuffle(clients)
     results = []
     number = 0
     for client in clients:
+        # dont add clients who have asked not to be displayed
+        if client.dont_display:
+            continue
```

Comment:
I know. The process will actually be more involved than that in order to move to WSGI, but that is one of several necessary steps (this pull request also moves the default centinel-server folder under /opt, so we will also need to create a directory with those permissions, manage the permissions, and migrate the data.)